test/clientTest.js

```diff
@@ -172,13 +200,14 @@ for (const supportedVersion of mc.supportedVersions) {
         })
       })
     })
-
-    describe('online', function () {
+    /*
```

Comment:
Why comment instead of skip ? It disables standard

Reply:
fixed in 8daf006